src/sentry/static/sentry/app/views/releasesV2/list/releaseHealth.tsx

```diff
@@ -186,7 +188,21 @@ const ReleaseHealth = ({release, orgSlug, location}: Props) => {
                     </CrashesColumn>
 
                     <NewIssuesColumn>
-                      <Count value={newGroups || 0} />
+                      {newGroups > 0 ? (
+                        <Tooltip title={t('Open in Issues')}>
+                          <Link
+                            to={getReleaseNewIssuesUrl(
+                              orgSlug,
+                              project.id,
+                              release.version
+                            )}
+                          >
+                            <Count value={newGroups || 0} />
```

Comment:
just a suggestion: I would rename newGroups to something like newGroupsQuantity

Reply:
It's coming from api